config/database.yml

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
Remove this file from the repository using the `git rebase` command and add it to the `.gitignore` file.